mailpoet/changelog/2025-12-16-12-18-50-fixed-shortcodes-in-urls-linking-the-sending-site-are-br.md

```diff
@@ -0,0 +1,5 @@
+# Type: Fixed
+
+# Description
+
+Shortcodes in URLs linking the sending site are broken
```

Comment:
I am not sure if this is understandable to someone who doesn't know anything about this issue. Could we use something more descriptive, like "Fixed shortcodes in URL parameters getting broken when GA tracking is enabled"